internal/cmd/build-uki.go

```diff
@@ -936,26 +936,21 @@ func createContainer(sourceDir, outputDir, artifactName, version string, logger
 // GetUkiCmdline returns the cmdline to be used for the kernel.
 // The cmdline can be overridden by the user using the cmdline flag.
 // For each cmdline passed, we generate a uki file with that cmdline
-// extend-cmdline will just extend the default cmdline so we only create one efi file
-// extra-cmdline will create a new efi file for each cmdline passed
+// extend-cmdline will just extend the default cmdline so we only create one efi file. Artifact name is the default one
+// extra-cmdline will create a new efi file for each cmdline passed. artifact name is generated from the cmdline
 func GetUkiCmdline(cmdlineExtend, bootBranding string, extraCmdlines []string) []utils.BootEntry {
 	defaultCmdLine := constants.UkiCmdline + " " + constants.UkiCmdlineInstall
 
-	// Extend only
+	// Override the default cmdline if the user passed one
 	if cmdlineExtend != "" {
-		cmdline := defaultCmdLine + " " + cmdlineExtend
-		return []utils.BootEntry{{
-			Cmdline:  cmdline,
-			Title:    bootBranding,
-			FileName: NameFromCmdline(constants.ArtifactBaseName, cmdline),
-		}}
+		defaultCmdLine = defaultCmdLine + " " + cmdlineExtend
```

Comment:
with this change, the extraCmdline on line 957, will also get the extended one, which wasn't the case up to now. I'm not sure if that's what we want or not. I'm just saying it's changing the behavior.